CHANGES.rst

```diff
@@ -20,6 +20,15 @@ Unreleased
 * :meth:`~.Emoji.update` supports modifying an emoji's permissions.
 * :meth:`~.SubredditEmoji.add` now supports optionally passing booleans to set
   an emoji's permissions upon upload.
+* There is a new method, :meth:`.Subreddit.post_requirements`, to fetch a
+  subreddit's post requirements.
+* Methods :meth:`.Subreddit.submit`, :meth:`.Subreddit.submit_image`,
+  :meth:`.Subreddit.submit_video`, and :meth:`.Submission.edit` all have a new
+  parameter, ``validate_on_submit``, which will cause Reddit to check that the
```

Comment:
Can you split these up into two separate PRs please?

Reply:
I feel that the new endpoint is a very minor feature, and there's really no need to create a separate PR just for it.

Reply:
It'll be quicker to review. If one is really small, then I'm likely to review it sometime this week.

Reply:
It has been split into this one and #1346